extensions/functions_arithmetic.yaml

```diff
@@ -153,6 +153,40 @@ scalar_functions:
           - value: fp64
           - value: fp64
         return: fp64
+  -
+    name: "negate"
+    description: "Negation of the value"
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
```

Comment:
Ugh... I suppose this exists because of the uneveness of twos-compliment but ugh...